src/services/expenses.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+let expenses = [];
+
+const init = () => {
+  expenses = [];
+};
+
+const getExpenses = () => {
+  return expenses;
+};
+
+const getFilteredExpenses = ({ userId, category, categories, from, to }) => {
+  let filteredExpenses = [...expenses];
+
+  if (userId) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.userId === +userId);
+  }
+
+  if (category) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.category === category);
+  }
+
+  if (categories) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => categories.includes(expense.category));
+  }
+
+  if (from) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.spentAt > from && expense.spentAt < to);
+  }
+
+  return filteredExpenses;
+};
+
+const getExpense = (expenseId) => {
+  return expenses.find(expense => expense.id === +expenseId);
+};
+
+const createExpense = (
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note,
+) => {
+  const newExpense = {
+    id: Math.random(),
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
```

Comment:
You can make your code shorter 
`const createExpense = (data) => {`
  `const newExpense = {`
   ` id: Math.random(),`
   ` ...data`
 ` };`